float32 battery_pct        # 0..100
float32 temperature_c
float32 gps_hacc_mm        # Horizontal Accuracy Estimate, mm
uint8   internet_status    # 0=no signal, 1=OK, 2=low signal
bool    estop              # true => emergency stop
